### DISCUSSION TITLE
HRRR conus data, Zarr format

### DISCUSSION THREAD

**Discussion by ganeshsarakadam** [blaylockbk/Herbie#123] [Q&A]
I'm working on the HRRR data (using Herbie ) and wanted to chunk the data using ZARR and store them in S3. 
I need some help to create chunks(Zarr chunks) from the xarray produced by Herbie and store those chunks in S3.
The AWS archive model currently has 2 hours delay from the UTC time. So can't really rely on that.

If there's anyone who could suggest something related to Zarr that would be really helpful.

**Comment by blaylockbk** [comment]
The HRRR model is already provided in zarr format on AWS (with about 3 hr delay), see here. https://blaylockbk.github.io/Herbie/_build/html/user_guide/notebooks/index.html#hrrr-in-zarr-format

You can reach out to the mesowest group if you have questions https://mesowest.utah.edu/html/hrrr/

**Reply by ganeshsarakadam** [reply]
I want to chunk the Herbie xarray dataset into a 150 x 150 shape using Zarr. 
Any suggestions on that?

**Reply by blaylockbk** [reply]
Sorry, I'm not familiar enough with zarr to provide advice on this.
